### DISCUSSION TITLE
v2.15.35

### DISCUSSION THREAD

**Discussion by danharrin** [filamentphp/filament#3770] [General]
## What's Changed
* Reverted extract dropdown by @zepfietje in https://github.com/filamentphp/filament/pull/3443
* Reverted support action groups in notifications by @zepfietje in https://github.com/filamentphp/filament/pull/3454

**Full Changelog**: https://github.com/filamentphp/filament/compare/v2.15.33...v2.15.35

<hr /><em>This discussion was created from the release <a href='https://github.com/filamentphp/filament/releases/tag/v2.15.35'>v2.15.35</a>.</em>

**Comment by lloricode** [comment]
It been nice to have small numeric for active filter, but seems it break something

**Reply by zepfietje** [reply]
What's broken, @lloricode?

**Reply by lloricode** [reply]
on last preview version 2.15.34 issue, and after this release its all fine now, im looking forward to that feature later
